src/test/java/bio/terra/common/BaseTest.java

```diff
@@ -0,0 +1,135 @@
+package bio.terra.common;
+
+import static org.junit.Assert.assertEquals;
+
+import bio.terra.cloudfunctions.common.MediaTypeWrapper;
+import com.google.cloud.functions.Context;
+import com.google.events.cloud.storage.v1.StorageObjectData;
+import java.io.InputStream;
+import org.junit.Before;
+
+public class BaseTest {
+  protected static String MOCK_EVENT_GZIP;
+  protected static String MOCK_EVENT_JSON;
+  protected static CFContext FAKE_CLOUD_FUNCTION_CONTEXT;
+  protected static MediaTypeWrapper APPLICATION_GZIP;
+  protected static MediaTypeWrapper APPLICATION_X_GZIP;
+  protected static MediaTypeWrapper APPLICATION_JSON;
+  protected static InputStream MOCK_TGZ;
+  protected static InputStream MOCK_GZ;
+  protected static InputStream MOCK_JSON;
+  protected static byte[] MOCK_STORAGE_OBJECT_DATASTREAM;
+
+  @Before
+  public void setUp() {
+    MOCK_EVENT_GZIP =
```

Comment:
Would it make sense to put these character-dense values in files? i.e. the values for `MOCK_EVENT_GZIP`, `MOCK_EVENT_JSON`, `MOCK_STORAGE_OBJECT_DATASTREAM` variables